tests/io/test_bc.py

```diff
@@ -98,7 +106,7 @@ class TestForcingBase:
         ],
     )
     def test_parses_function_case_insensitive(self, input, expected):
-        forcing = ForcingBase(function=input, name="somename", quantity=[], unit=[])
+        forcing = ForcingBase(function=input, name="somename", quantities=[])
```

Comment:
In some earlier commit, apparently ```quantity``` was renamed to ```quantities```.
This intuitive, but it is inconsistent with class members in many other classes, for example: ```StructureModel.structure```, ```CrossDefModel.definition```, etc., which are all also lists.
Not sure what is best to change or keep here...

Reply:
Changed the name as agreed to `quantityunitpair`